src/Microsoft.AspNet.OData/Microsoft.AspNet.OData.csproj

```diff
@@ -7,8 +7,9 @@
     <RootNamespace>Microsoft.AspNet.OData</RootNamespace>
     <AssemblyName>Microsoft.AspNet.OData</AssemblyName>
     <DocumentationFile>$(OutputPath)$(AssemblyName).xml</DocumentationFile>
-    <RunCodeAnalysis>$(CodeAnalysis)</RunCodeAnalysis>
+    <RunCodeAnalysis Condition=" '$(CodeAnalysis)' == '' and '$(Configuration)' != 'Release' ">true</RunCodeAnalysis>
```

Comment:
In case anyone else is wondering about the != 'Release', here's a snippet from a different part of the build script.
"Unable to run code analysis in Release configuration. Release assemblies do not include SuppressMessage attributes (so code analysis would always fail with the errors that are normally suppressed)."